org-code-javabuilder/media/src/main/java/org/code/media/Image.java

```diff
@@ -1,16 +1,31 @@
 package org.code.media;
 
+import java.awt.image.BufferedImage;
 import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.net.URL;
+import javax.imageio.ImageIO;
+import org.code.protocol.GlobalProtocol;
 
 public class Image {
+  private BufferedImage bufferedImage;
+
   /**
    * Creates a new image object, using the pixel information from the file uploaded to the asset
    * manager.
    *
    * @param filename the name of the image loaded into the asset manager for the project
    * @throws FileNotFoundException if the file doesn't exist in the asset manager.
    */
-  public Image(String filename) throws FileNotFoundException {}
+  public Image(String filename) throws FileNotFoundException {
+    try {
+      this.bufferedImage =
+          ImageIO.read(new URL(GlobalProtocol.getInstance().generateAssetUrl(filename)));
+    } catch (IOException e) {
+      // TODO: improve error handling
+      throw new FileNotFoundException();
```

Comment:
Mind adding a task for this?

Reply:
this is going to be a part of the task to implement the theater image api, this was a placeholder for testing this feature